addons/fortify/initSettings.sqf

```diff
@@ -9,3 +9,17 @@
         2
     ]
 ] call CBA_settings_fnc_init;
+[
+    QGVAR(markObjectsOnMap),
+    "LIST",
+    [LLSTRING(markObjectsOnMap), LLSTRING(markObjectsOnMapDesc)],
+    LLSTRING(settingsCategory),
+    [
+        [0, 1, 2],
+        [LLSTRING(markObjectsOnMapNone), LLSTRING(markObjectsOnMapFriendly), LLSTRING(markObjectsOnMapEveryone)],
+        1
+    ],
+    true,
+    {},
+    true
+] call CBA_settings_fnc_init;
```

Comment:
```suggestion
] call CBA_fnc_addSetting;
```

Reply:
Same file is also using CBA_settings_fnc_init, isn't mixing & matching worse?

Reply:
Can also change the others, here or in separate PR. addSetting is public API, init is old.